src/applications/resources-and-support/hooks/useGetSearchResults.js

```diff
@@ -43,7 +44,44 @@ export default function useGetSearchResults(articles, query, page) {
         });
       });
 
-      const orderedResults = sortBy(filteredArticles, 'title');
+      let orderedResults = [];
+
+      // Keep legacy sorting on production.
+      if (environment.isProduction()) {
+        orderedResults = orderBy(filteredArticles, 'title');
+
+        // Experiment with new sorting on non-prod envs.
+      } else {
+        filteredArticles = filteredArticles?.map(article => ({
+          ...article,
+
+          // Number of times a keyword is found in the article's title.
+          keywordsCountsTitle: keywords?.reduce((keywordInstances, keyword) => {
+            keywordInstances +=
+              article.title.toLowerCase()?.split(keyword)?.length || 0;
+            return keywordInstances - 1;
+          }, 0),
+
+          // Number of times a keyword is found in the article's description.
+          keywordsCountsDescription: keywords?.reduce(
+            (keywordInstances, keyword) => {
+              keywordInstances +=
+                article.description.toLowerCase()?.split(keyword)?.length || 0;
+              return keywordInstances - 1;
```

Comment:
This could return a -1 if your description doesn't contain the keyword. Just wanted to make sure this was the intended functionality?

Reply:
Oh good catch! 💯 Updating this but I don't think it's that much of a concern what the min keyword instance number is, just as long as it is accurate with respect to the surrounding articles 🙂 Regardless, that's confusing so I'll update it now!